fem/bilininteg.hpp

```diff
@@ -3771,6 +3864,15 @@ class NormalTraceJumpIntegrator : public BilinearFormIntegrator
                            FaceElementTransformations &Trans,
                            DenseMatrix &elmat) override;
 
+   void AssembleTraceFaceMatrix(int ielem,
```

Comment:
Do we need this new function that just wraps `AssembleFaceMatrix` with unused argument `ielem`?